README.md

```diff
@@ -145,7 +145,7 @@ Live monitoring dashboard
 
 ```
 export MONITOR=true
-stack run fission-web
+stack run
```

Comment:
`stack` no longer has to disambiguate, so `stack run` is sufficient. We have the `Makefile`, too, so this won't matter to the internal team too much.